src/style.css

```diff
@@ -0,0 +1,167 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;1,100;1,300;1,400&display=swap');
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+  width: 100%;
+  height: 100vh;
+}
+
+ul {
+  list-style: none;
+}
+
+input,
+button {
+  border: none;
+  outline: none;
+  background-color: transparent;
+}
+
+i {
+  cursor: pointer;
+}
+
+.flex-center {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+#tasks {
+  width: 100%;
+  height: 100%;
+  flex-direction: column;
+}
+
+.task-list {
+  flex-direction: column;
+  width: 40%;
+  box-shadow: 0 3px 10px rgb(0 0 0 / 0.2);
+  border-radius: 0.5rem;
+}
+
+.task-list .container {
+  flex-direction: row;
+  padding: 1rem 0.9rem;
+  width: 100%;
+  border-bottom: 1px solid rgb(221, 219, 219);
+  justify-content: space-between;
+}
+
+.task-list-container {
+  width: 100%;
+}
+
+.task-list #add-task {
+  width: 100%;
+  justify-content: space-between;
+}
+
+.task-list #add-task button {
+  padding: 0.6rem;
+  background-color: transparent;
+}
+
+.task-list-container .task i {
+  font-size: 1rem;
+  color: rgb(102, 99, 99);
+}
+
+.task-list .container:first-child i {
+  font-size: 1rem;
+  cursor: pointer;
+}
+
+.task-list #add-task button i {
+  font-size: 0.8rem;
+  color: #ccc;
+  transform: rotate(90deg);
+}
+
+.task-list #add-task button i:hover {
+  color: rgb(104, 100, 100);
+}
+
+.task-list #add-task input {
+  flex: 1;
+  font-family: 'Roboto', sans-serif;
+  font-style: italic;
+  font-size: 1rem;
+  font-weight: 300;
+}
+
+.task-list-container .task {
+  width: 100%;
+  justify-content: space-between;
+  padding: 1rem 0.9rem;
+  cursor: move;
+}
+
+.task-list-container .task .left {
+  width: 90%;
+}
+
+.task-list-container .task .left form {
+  width: 100%;
+  display: flex;
+}
+
+.task-list-container .task .left .edit-task {
+  padding-left: 0.7rem;
+  font-size: 1rem;
```

Comment:
I suggest assigning a global value for font size directly instead of reassigning the value separately in multiple places. This way, you can save a few lines of code and avoid repetition.